package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "docs",
+  "version": "1.0.0",
+  "description": "A central repository for documentation",
+  "main": "README.md",
+  "dependencies": {
+    "markdown-link-check": "^3.8.1",
+    "markdown-spellcheck": "^1.3.1",
+    "markdownlint": "^0.20.3",
+    "markdownlint-cli": "^0.23.1"
+  },
+  "devDependencies": {},
+  "scripts": {
+    "test": "make check"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:corona-warn-app/cwa-documentation.git"
+  },
+  "keywords": [
+    "docs"
+  ],
+  "author": "Johannes Amorosa",
```

Comment:
This needs to be addressed. What should be in here?

Reply:
Let's go with docs for now. That's already 1 keyword more than we have now ;)